CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# ChangeLog for FreeRTOS
+
+## V200717_LTS_development_snapshot July 2020
+
+### New Features
+
+#### Over the Air Update
+
+- Updated the over-the-air (OTA) agent with the ability to pause and resume an in-progress update.
+- Updated the OTA demo to demonstrate how to suspend an in-progress OTA update should the MQTT connection disconnect, then resume the same update when the MQTT connection reconnects. In line with best practice, the reconnect logic uses an exponential backoff and jitter algorithm to prevent the MQTT server getting overwhelmed with connection requests if a fleet of devices get disconnected and then attempt to reconnect at the same time.
+- For test purposes only, it is now possible to use OTA to downgrade the version number.  That capability is disabled by default.
+- Add doxygen comments to the OTA source files.
+
+### Updates
+
+#### Rename FreeRTOS-IoT-Libraries Directory
+
+- The FreeRTOS-IoT-Libraries directories in the previous snapshot version (V2020218_LTS_development_snapshot) under FreeRTOS-Plus/Source and FreeRTOS-Plus/Demo has been renamed to *FreeRTOS-IoT-Libraries-Beta1* to make way for a parallel directory, *FreeRTOS-IoT-Libraries-Beta2*, that holds libraries for which the refactoring work is near completion (currently the MQTT library).
+
+#### New MQTT Client Library (Beta 2)
+
+- A new MQTT library source code has been added under the FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-Beta2 directory.
```

Comment:
Should we use refactored or new. I thought we decided to use 'refactored' .

Reply:
We decided to use _refactored_ only for the "What's New Post" but Beta2 elsewhere to match the directory containing the new library. 
As the section header contains "New", it would be fine to use that in the beginning of the sentence

Reply:
My comment was for both the header and the content. Have we confirmed this with Richard ?

Reply:
Yes, the use of "Beta 2" has been suggested by @RichardBarry.